packages/runtime-vapor/src/component.ts

```diff
@@ -20,11 +21,69 @@ export interface ComponentInternalInstance {
 
   component: FunctionalComponent | ObjectComponent
   get isMounted(): boolean
+  get isUnMounted(): boolean
```

Comment:
It seems that in vuejs/core, 'unmount' is treated as a single word, so I think 'isUnmounted' would be better.